### ISSUE TITLE
call a function when input box is clicked

### ISSUE THREAD

**Issue by c0debreaker** [angular-ui/ui-select#1133]
I added ng-focus="clearModel()" inside ui-select but it didn't call it. I moved it to ui-select-match which you see below, it still didn't call clearModel() function. What am I doing wrong? I'm doing it this way because i have different dropdowns that are related to each other.

```
        <ui-select ng-model="model.selected" ng-disabled="disabled">
          <ui-select-match ng-focus="clearModel()" placeholder="enter name...">{{$select.selected.name}}</ui-select-match>
          <ui-select-choices repeat="model in displayedCollection | propsFilter: {name: $select.search}">
            <div ng-bind-html="model.name | highlight: $select.search"></div>
          </ui-select-choices>
        </ui-select>
```

**Comment by carlosfunk** [comment]
Is the uis-open-close the solution to this?  Available in 0.19.0 https://github.com/angular-ui/ui-select/wiki/uis-open-close

**Comment by c0debreaker** [comment]
I'm closing this for now. I've forgotten how I got it to work.
